examples/fabric-contract-example-gradle-kotlin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-bin.zip
```

Comment:
Why has this changed to a back level version?

Reply:
good spot - mismerge...